dags/bqetl_main_summary.py

```diff
@@ -119,7 +144,7 @@
         task_id="wait_for_copy_deduplicate_copy_deduplicate_main_ping",
         external_dag_id="copy_deduplicate",
         external_task_id="copy_deduplicate_main_ping",
-        execution_delta=datetime.timedelta(seconds=3600),
+        execution_delta=datetime.timedelta(seconds=7200),
```

Comment:
This needs to be an hour `execution_delta`.

```suggestion
        execution_delta=datetime.timedelta(seconds=3600),
```

Probably merged https://github.com/mozilla/bigquery-etl/pull/1160 after you generated these DAGs.